docs/content/extras/urls.md

```diff
@@ -61,3 +61,12 @@ Or, if you are on Windows and do not have `grep` installed:
 
     hugo config | FINDSTR /I canon
 
+## Relative URLs
+
+By default, all relative URLs are left unchanged by Hugo, but setting
+`relativeURLs` to `true` in the site configuration will cause Hugo to rewrite
+all relative URLs to be relative to the current content.
+
+For example, if the `/post/first/` page contained a link with a relative
+URL of `/about/`, Hugo would rewrite that URL to `../../about/`.
```

Comment:
It would be nice with a line about the main use case for this feature: Having Hugo sites browse-able from the file system, documentation that can be zipped and sent via mail etc.